glusterd2/volgen2/store-utils.go

```diff
@@ -2,10 +2,11 @@ package volgen2
 
 import (
 	"context"
-
-	"github.com/gluster/glusterd2/glusterd2/store"
+	"strings"
 
 	"github.com/coreos/etcd/clientv3"
+	"github.com/gluster/glusterd2/glusterd2/store"
```

Comment:
We follow a convention for grouping imports. See **Imports:** section from this document: https://github.com/gluster/glusterd2/blob/master/doc/coding.md